### DISCUSSION TITLE
List of other v1 proxies

### DISCUSSION THREAD

**Discussion by younesaassila** [younesaassila/ttv-lol-pro#37] [General]
Share other proxies that are compatible with TTV LOL PRO v1.

**Comment by younesaassila** [comment]
- `https://eu.luminous.dev` (Europe)
- `https://as.luminous.dev` (Asia)

by @AlyoshaVasilieva

**Reply by thinkpad4** [reply]
`https://eu2.luminous.dev` (Europe 2)

Also by @AlyoshaVasilieva

**Reply by revunix** [reply]
Asia works very good, no ads since weeks.

**Reply by MisterSheeple** [reply]
EU working fine for me, while the TTV LOL API is seeing outages at the moment.

**Reply by AitoZ** [reply]
`https://bg.luminous.dev` (Bulgaria)

Also by @AlyoshaVasilieva

**Reply by thinkpad4** [reply]
`https://bg.luminous.dev` Doesn't seem to be working

**Reply by AlyoshaVasilieva** [reply]
I haven't updated the Bulgarian server in months, it's running on a very weak, out-of-date VPS

**Reply by AlyoshaVasilieva** [reply]
Due to imminent expiry of a subscription, replaced outbound on `as` (should still be detected by Twitch as China) and `eu` (should now be detected as Russia). `eu2` may be down for a bit but it has far fewer users

**Comment by zGato** [comment]
- `https://lb-eu.cdn-perfprod.com` (Europe)
- `https://lb-eu2.cdn-perfprod.com` (Europe 2)
- `https://lb-eu4.cdn-perfprod.com` (Europe 4)
- `https://lb-eu5.cdn-perfprod.com` (Europe 5)
- `https://lb-na.cdn-perfprod.com` (NA)
- `https://lb-as.cdn-perfprod.com` (Asia)
- `https://lb-sa.cdn-perfprod.com` (SA)

Using [LuminousTTV](https://github.com/AlyoshaVasilieva/luminous-ttv) server backend

The following proxies are more useful for mobile users (as they don't have stitched ads, the V1 system still works for them)
- `https://lb-eu3.cdn-perfprod.com` (Europe 3, using Russia-only proxies) (Mobile & PC compatible, using [LuminousTTV](https://github.com/AlyoshaVasilieva/luminous-ttv))

To use this proxy with Twitch modded apps you need to specify in the address field the following: `https://lb-eu3.cdn-perfprod.com/live/$channel?allow_source=true&allow_audio_only=true&fast_bread=true` (Tested with Xtra app and works!)
`$channel` is the variable for the channel name. Change it if your app uses other variable.

I've removed the PurpleTV reverse-proxy as their server has been down for a while now.

Status Page: https://status.cdn-perfprod.com
by @zGato (me)

**Reply by ipsusu** [reply]
Thank you! I was having issues with the default proxies but swapping to this seemed to fix my loading problems.

**Comment by sackramento** [comment]
What is Purple Adblock's proxy server?

**Reply by zGato** [reply]
Not compatible with this extension.

**Comment by SakhG** [comment]


**Reply by MisterSheeple** [reply]
What proxy are you trying to use?

**Comment by MuhannadRidah** [comment]
i get ads on EU proxy , no ads on NA/Asia proxy but its laggy

**Comment by CarolineLightning** [comment]
None of the proxies work anymore. Worked for almost 2 months before. 
Any clue why?

**Reply by thinkpad4** [reply]
They all work fine

**Reply by eldepor** [reply]
> They all work fine

How to check if they working or not when watching tw?

**Reply by zGato** [reply]
You can test them by clicking on the extension icon while watching a stream or access them with /ping
Ex: https://lb-eu2.perfprod.com/ping

**Comment by Makezu90** [comment]


**Reply by MisterSheeple** [reply]
Haven't tested AS, but no ads on the EU server.

**Reply by gruberdev** [reply]
Me too. Switched to EU server and got no ads after that.

**Comment by gangchains** [comment]
brazil/austrilian//south african/fcking southpole proxies when? need the most obsecure places to house proxies to fight the arms race its only a matter of time before eu is depracated just like as/na... unlucky i guess :/

**Reply by ghost** [reply]
yeah... guess I gotta enjoy the proxies while they last huh? Before I have to go back to the player I was using that autopaused to filter out ads.

**Reply by zGato** [reply]
This doesn't work as you think it does. Twitch is rolling an update which makes this ad blocking method not work in some countries. If none proxies posted here works for you, no other will work for you aswell even if it's in Antarctica.

I needed to remove the RU proxies from the EU2 load balancer because they gave ads. I do have proxies in more countries, it will work until the patch everything. Try using that, or AS ones

**Reply by thinkpad4** [reply]
Thank you sir

**Comment by filipemanuelofs** [comment]
`https://as.luminous.dev` doesn't seem to be working properly, I recently noticed these ads. I'm going to switch to the EU.

![image](https://github.com/younesaassila/ttv-lol-pro/assets/1047292/83167def-5d7e-4627-8eb0-8960050a61fc)

**Comment by zodmaster** [comment]
I'm getting ads, with all the list of proxies
 
![image](https://github.com/younesaassila/ttv-lol-pro/assets/20826330/feb64485-643f-4c17-9248-3685f65b25a5)

I am changing one by one, but I always see ads
![image](https://github.com/younesaassila/ttv-lol-pro/assets/20826330/d1076ff7-43ce-4b95-b19d-89d826e77250)


Any advice? please. Thank you for your effort in sharing your time and knowledge!

**Reply by zGato** [reply]
Please provide me your discord user. I'll be adding you to a group to do some testing if you'd like. Also, you need to delete browser cache for your ad to change from this cat to another, and maybe it works then.

**Reply by zGato** [reply]
Added you.

**Reply by Jacubz** [reply]
honestly if the proxies are starting to stop working or are showing ads, chances are either ttv-lol pro needs to start looking for new proxy servers because it looks like twitch are bypassing. that is if twitch isn't beginning to crack down on the extension altogether.

**Comment by fabcard** [comment]


**Reply by filipemanuelofs** [reply]
Try the EU version: `https://eu.luminous.dev`

**Comment by Maxterino** [comment]
Hello, the as.luminous.dev proxy worked perfectly for the last 3 months till now. Im in The Netherlands, Europe. How can i fix this? Is there a different proxy that works well for my region?

**Reply by AlyoshaVasilieva** [reply]
Try https://eu.luminous.dev or one of the proxies in https://github.com/younesaassila/ttv-lol-pro/discussions/37#discussioncomment-5426032

as.luminous.dev is down due to DDoS (aimed at datacenter, not me)

edit: should be up now, DDoS seems to have stopped

**Reply by thinkpad4** [reply]
@AlyoshaVasilieva Will you be creating a new proxy for the new 2.0 version of TTVLOLPRO?

**Reply by AlyoshaVasilieva** [reply]
@thinkpad4 I am not yet affected by Twitch's changes, so testing it out currently would be annoying. I'll try to get to it within a week.

**Comment by ynnniV** [comment]
Someone got a proxy who work? Because no one is working atm.

**Reply by juliocoliveira** [reply]
https://eu.luminous.dev is working fine, shows an ad very rarely, I'm using it right now without any ads.

**Reply by zGato** [reply]
Just switch and give feedback on the new beta version ;)

**Reply by deprale** [reply]
eu luminous ad 7 of 7 :)

**Reply by ynnniV** [reply]
at the moment, no one is working! Sad!

**Reply by thinkpad4** [reply]
> eu luminous ad 7 of 7 :)

Try eu2 luminous

**Reply by loopy750** [reply]
I've noticed nothing seems to work properly between roughly 7-9pm UTC.

**Comment by M1guelinn** [comment]
None of them work for me, I want to try the tests, discord: Miguelín#6945

**Reply by zGato** [reply]
Added you.

**Comment by Andore97** [comment]
yeah the ads are stacking for me.. no proxy is working.

**Reply by younesaassila** [reply]
Could you try out the v2 beta and let us know how it works for you? Thanks
https://github.com/younesaassila/ttv-lol-pro/releases

**Reply by ynnniV** [reply]
The v2 is working super well! Nice job guys!

**Reply by Maxterino** [reply]
> Could you try out the v2 beta and let us know how it works for you? Thanks https://github.com/younesaassila/ttv-lol-pro/releases

Do i still need to use custom proxy's or are the stable.ttvlol and asia.ttvlol proxy's enough?

**Reply by ynnniV** [reply]
> > Could you try out the v2 beta and let us know how it works for you? Thanks https://github.com/younesaassila/ttv-lol-pro/releases
> 
> Do i still need to use custom proxy's or are the stable.ttvlol and asia.ttvlol proxy's enough?

I'm using the v2 without the standard proxies and is working super good!

**Comment by kittykkuma** [comment]
if anyone has documentation about setting up a proxy for ttvlol i would be more than happy to get them up.

**Comment by jeffvffv** [comment]
![image](https://github.com/younesaassila/ttv-lol-pro/assets/113591568/935544fd-cb97-4a98-9270-6a7cf1d8df14)
ads on default proxy

**Reply by younesaassila** [reply]
Yeah default proxy is not that great

**Reply by jeffvffv** [reply]
but the custom proxy buffering and laggy for me even the asia proxy

**Comment by ItsBluey** [comment]
None of them work for me, I want to try the tests, discord: Bluey#7154

**Reply by revunix** [reply]
Same here, not even my own server works.

**Reply by schrobby** [reply]
Same for me. All proxies stopped working a little over an hour ago. 
I switched to V2 beta 8 with the proxy for my region listed in #130 and it resolved this issue entirely:

**Reply by ItsBluey** [reply]
> Same for me. All proxies stopped working a little over an hour ago. I switched to V2 beta 8 with the proxy for my region listed in #130 and it resolved this issue entirely:
> 
> ```
> beta.ttvlolpro.perfprod.com:4019 (EU)
> america.ttvlolpro.perfprod.com:8014 (NA)
> ```

im on chrome and beta doesnt work for it yet

**Reply by zGato** [reply]
Stop rolling out proxies IP everywhere. Just point them to the discussions page where I explain all.
The proxies you guys mentioned are shut down. Read here to get the new ones https://github.com/younesaassila/ttv-lol-pro/discussions/130#discussioncomment-5921097

**Reply by schrobby** [reply]
> Stop rolling out proxies IP everywhere. Just point them to the discussions page where I explain all. The proxies you guys mentioned are shut down. Read here to get the new ones [#130 (comment)](https://github.com/younesaassila/ttv-lol-pro/discussions/130#discussioncomment-5921097)

Updated my comment.

**Comment by Zachruff** [comment]
discord: Zach#2169 having issues now, want to try the new beta

**Comment by zGato** [comment]
To everyone still using V1 perfprod.com proxies, please update your Proxy URLs to the following

- From `https://lb-eu.perfprod.com` to `https://lb-eu.cdn-perfprod.com`
- From `https://lb-eu2.perfprod.com` to `https://lb-eu2.cdn-perfprod.com`
- From `https://lb-na.perfprod.com` to `https://lb-na.cdn-perfprod.com`
- From `https://lb-as.perfprod.com` to `https://lb-as.cdn-perfprod.com`

Sorry for the inconveniences, but this change was needed. Rest assured there won't be any more changes.
Thanks for your understanding

With ❤️ from @zGato 

(We've also updated recently the NA proxy, you'll see a HUGE improvement if you're from there and you use it)

Also, huge thanks to @AlyoshaVasilieva for his amazing work on [LuminousTTV](https://github.com/AlyoshaVasilieva/luminous-ttv). These proxies wouldn't be possible without his work. If you ever need servers for luminous.dev proxies, hit me up :)

**Reply by thinkpad4** [reply]
@zGato You should either make a new post explaining this change or, if you can, pin this to the top of this post

**Reply by 21mtd** [reply]
these proxies are down for me today

**Reply by zGato** [reply]
holy shit mate, 90d with barely downtime, I do a little maintenace to apply security/enchancement updates and they're down "today"
![image](https://github.com/younesaassila/ttv-lol-pro/assets/88756568/a4cac7fc-eeb1-46de-8055-2ec2ab95f0d4)

They'll be up once they're stable back again. There's a status page for them for a reason. https://status.cdn-perfprod.com

**Reply by 21mtd** [reply]
i didnt mean anything just want to inform you in case you didnt know, thanks for the update

**Comment by Fijxu** [comment]
New RU proxy at https://twitch.nadeko.net/ using luminous-ttv. It is actually hosted on the US but it returns a RU playlist :p

**Reply by zGato** [reply]
Just curious, which provider are you using to host the luminous-ttv instance? (not your US reverse-proxy)

**Reply by Fijxu** [reply]
Proxy6 shared RU proxy

**Reply by PeterDLai** [reply]
Did Twitch change something that sometimes causes this to fail? I am noticing French ads occasionally.

**Reply by zGato** [reply]
V1 is deprecated and not supported.

**Reply by SakhG** [reply]
720p only, since Twitch restricted Russia

**Reply by david-UT99** [reply]
twitch killed yours and the 'luminous.dev' ones today. show up as offline, then might start after a minute or 2 with mini screen in chat.

(also broke adguard's twitch script/addon)

**Reply by Fijxu** [reply]
Hi, I'm sure there is still some people using the TTVLOL v1 proxy as it still works fine using https://github.com/2bc4/streamlink-ttvlol. I recently got a server in albania so since ads in albania are not allowed, I decided to host another ttvlol v1 instance at https://twitch-al.nadeko.net, feel free to use it! It offers 1080p compared to the Russia one.

**Reply by thinkpad4** [reply]
@Fijxu I can't seem to get this to work with Xtra app
https://twitch-al.nadeko.net/live/$channel?allow_source=true&allow_audio_only=true&fast_bread=true

I tried that and just a spinning circle

**Reply by Fijxu** [reply]
> @Fijxu I can't seem to get this to work with Xtra app https://twitch-al.nadeko.net/live/$channel?allow_source=true&allow_audio_only=true&fast_bread=true
> 
> I tried that and just a spinning circle

The problem seems to be with Xtra, as they say `"$channel" will be replaced with channel login` in the settings, but the channel login ends up being `fijxu.m3u8` instead of `fijxu`, so the URL Xtra crafted ends up being wrong :/

Xtra does: `https://twitch-al.nadeko.net/playlist/fijxu.m3u8?allow_source=true&allow_audio_only=true&fast_bread=true` (wrong!) instead of `https://twitch-al.nadeko.net/playlist/fijxu?allow_source=true&allow_audio_only=true&fast_bread=true` (Good!)

**Reply by thinkpad4** [reply]
@Fijxu Is there a way to get it to work?

**Reply by Fijxu** [reply]
> @Fijxu Is there a way to get it to work?

Report issue to Xtra

**Reply by thinkpad4** [reply]
@Fijxu Got it.   Also, thank you for hosting this proxy, it works fine on Chrome/Firefox on the desktop.   <3

**Reply by thinkpad4** [reply]
@Fijxu I got it working, I needed to update the Xtra app.   <3
